cthreads.c

```diff
@@ -495,3 +495,80 @@ int cthreads_cond_timedwait(struct cthreads_cond *cond, struct cthreads_mutex *m
     return platform_error_str_len + 1;
   }
 #endif
+
+#ifdef CTHREADS_SEMAPHORE
+  int cthreads_sem_init(struct cthreads_semaphore* sem, int initial_count){
```

Comment:
```suggestion
  int cthreads_sem_init(struct cthreads_semaphore *sem, int initial_count) {
```

Same for other definitions